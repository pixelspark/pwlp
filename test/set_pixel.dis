0000.	31	PUSHI	[05, cc, 00, 00]
0005.	e3	USER	set_pixel
0006.	01	POP	1
0007.	e4	USER	blit

