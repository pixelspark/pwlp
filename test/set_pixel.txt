set_pixel(0x05|0xCC00);
blit;
