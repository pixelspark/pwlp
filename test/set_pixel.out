000000 000000 000000 000000 000000 cc0000 000000 000000 000000 000000 
